pxtblocks/fields/field_note.ts

```diff
@@ -185,7 +235,10 @@ namespace pxtblockly {
          * @param {string} note The new note in string format.
          */
         setValue(note: string) {
-            note = String(parseFloat(note || "0"));
+            let regex: RegExp = new RegExp("Note.(.*)");
+            let match = regex.exec(note);
+            let noteName: any = (match && match.length > 1) ? match[1] : null;
+            note = Note[noteName] ? String(parseFloat(Note[noteName])) : String(parseFloat(note || "0"));
```

Comment:
The first `parseFloat` call here isn't needed. I think it should already be a number